jetson/nav/obstacle_avoidance/obstacleAvoidanceStateMachine.cpp

```diff
@@ -68,7 +68,9 @@ NavState ObstacleAvoidanceStateMachine::run()
 bool ObstacleAvoidanceStateMachine::isTargetDetected ()
 {
     return ( mRover->roverStatus().currentState() == NavState::SearchTurnAroundObs &&
-             mRover->roverStatus().leftTarget().distance >= 0 );
+             ( mRover->roverStatus().leftTarget().distance >= 0 || 
+                ( mRover->roverStatus().leftTarget().distance < 0 && 
```

Comment:
Is this second check for when you see the tag, but don't get an accurate distance? Could you add a comment explaining that if so?

Reply:
^